modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -488,26 +488,48 @@ bool GDMono::_load_project_assembly() {
 #endif
 
 #ifdef GD_MONO_HOT_RELOAD
+void GDMono::reload_failure() {
+	if (++project_load_failure_count >= static_cast<int64_t>(GLOBAL_GET("dotnet/project/assembly_reload_attempts"))) {
```

Comment:
This seems to be what we do in other usages of `GLOBAL_GET`:

```suggestion
	if (++project_load_failure_count >= (int)GLOBAL_GET("dotnet/project/assembly_reload_attempts")) {
```

But take it with a grain of salt, I'm not well versed in C++.

Reply:
`static_cast` is the "correct" c++ way of calling conversion functions like this, `(type)` are C style casts, which "should" generally be avoided as it can accidentally cast away `const` and it is not always clear how the cast is supposed to work.

Tho even when using c style casts, `int64_t` is the correct type or you risk accidental overflows (Variant stores 64 bit ints)

Reply:
In this case, we value consistency over absolute correctness. There's no risk of casting away `const` here so a C style cast is fine and the most common usage in the Godot codebase.